src/storage/CookiesStorage.js

```diff
@@ -47,7 +47,7 @@ class CookiesStorage {
   /**
    * Finds the registrable domain of current URL and sets it as the default
    *    domain for cookies
-   * @param {!document} document the document whose cookies are to be set.
+   * @param {!Object} document the document whose cookies are to be set.
    * @return {?string} the registrable domain of docDomain
    * @private
```

Comment:
Nit: The 3 sentences in this JSdoc should start with a capital letter and end in a period. Also check the rest of this file